fastlane/Fastfile

```diff
@@ -37,6 +37,59 @@ platform :android do
     bump(options)
     attach_changelog_to_master
   end
+
+  desc "Make github release"
+  lane :github_release do |options|
+    release_version = options[:version]
+    fail ArgumentError, "missing version" unless release_version
+
+    begin
+      changelog = File.read("../CHANGELOG.latest.md")
+    rescue
+      fail "please add a CHANGELOG.latest.md file before calling this lane"
+    end
+
+    set_github_release(
+      repository_name: "revenuecat/purchases-android",
+      api_token: ENV["GITHUB_TOKEN"],
+      name: release_version,
+      tag_name: "#{release_version}",
+      description: changelog,
+      commitish: "master",
+      upload_assets: [],
+      is_draft: true
```

Comment:
you probably don't need the draft thing, I've been meaning to remove it from iOS

Reply:
(I realize that this code was only moved)